covid19-etl/helper/file_helper.py

```diff
@@ -0,0 +1,63 @@
+import requests
+
+
+def upload_file(path, url):
+    with open(path, "rb") as data:
+        r = requests.put(url, data=data)
+        print("status: {}\ncontent: {}".format(r.status_code, r.content))
+        return r.status_code
+
+
+class FileHelper:
+    def __init__(self, base_url, program_name, project_code, access_token):
+        self.base_url = base_url
+        self.program_name = program_name
+        self.project_code = project_code
+        self.headers = {"Authorization": f"Bearer {access_token}"}
+
+    def find_by_name(self, filename):
+        url = f"{self.base_url}/index/index?file_name={filename}"
```

Comment:
i don't really like the idea of finding a record by filename because it's not supposed to be unique, so when we add these tools to the SDK, let's not add this one